addons/nuxeo-csv/ftest/features/step_definitions/csv-import.js

```diff
@@ -1,18 +1,29 @@
 import { When } from 'cucumber';
 
-When(/^I go to the (.+) tab$/, function(name) {
+/**
+ * Import the csv file
+ *
+ * @deprecated since 3.0.0. Please use "I upload the (.+) on the tab content page" located in create_dialog.js
```

Comment:
3.0.0 was released a long time ago. Next release will be 3.0.3 (hopefully next week)
```suggestion
 * @deprecated since 3.0.3. Please use "I upload the (.+) on the tab content page" located in create_dialog.js
```